.github/workflows/deploy.yml

```diff
@@ -0,0 +1,145 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Our mainline branch is main, not master

Reply:
This is Github Co-pilot giving me the wrong hints and me forgetting we use master lol